__tests__/app.tests.js

```diff
@@ -74,37 +107,75 @@ describe("/api/articles/:articles_id", () => {
         expect(response.body.msg).toBe("bad request");
       });
   });
-});
-
-describe("/api/articles", () => {
```

Comment:
You might also look to add a test for 
Status 200, if inc_votes is missing, should not update article and return original single article object (this may be included as a 400 but that could make for a brittle test suite)